perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_resource_attribute_def_def_o365EmailAddresses_mu.java

```diff
@@ -43,19 +46,22 @@
 @SuppressWarnings("unused")
 public class urn_perun_group_resource_attribute_def_def_o365EmailAddresses_mu extends ResourceGroupAttributesModuleAbstract implements ResourceGroupAttributesModuleImplApi {
 
+	private final static Logger log = LoggerFactory.getLogger(urn_perun_group_resource_attribute_def_def_o365EmailAddresses_mu.class);
+
 	private static final String NAMESPACE = AttributesManager.NS_GROUP_RESOURCE_ATTR_DEF;
 	static final String ADNAME_ATTRIBUTE = NAMESPACE + ":adName";
 
 	@Override
 	public void checkAttributeValue(PerunSessionImpl sess, Resource resource, Group group, Attribute attribute) throws InternalErrorException, WrongAttributeValueException, WrongReferenceAttributeValueException, WrongAttributeAssignmentException {
-		List<String> emails;
+		log.trace("checkAttributeValue(resource={},group={},attribute={})",resource,group,attribute);
```

Comment:
This was probably only for testing purpose, do you need it in production code too?